data/presets/landuse/meadow_orchard/orchard.json

```
@@ -0,0 +1,37 @@
+{
+    "icon": "maki-park",
+    "fields": [
+        "name",
+        "operator",
+        "trees"
+    ],
+    "moreFields": [
+        "{@templates/contact}",
+        "address",
+        "species/wikidata",
+        "surface",
+        "description"
+    ],
+    "geometry": [
+        "area"
+    ],
+    "tags": {
+        "landuse": "orchard",
+        "orchard": "meadow_orchard"
+    },
+    "reference": {
+        "key": "orchard",
+        "value": "meadow_orchard"
+    },
+    "terms": [
+        "fruit",
+        "maedow"
+    ],
+    "name": "Orchard with fruit trees",
```

does a (regular) orchard not always have fruit trees? Shouldn't this preset have something like

```suggestion
    "name": "Orchard Planted in a Meadow",
```